cspell.json

```diff
@@ -22,45 +22,50 @@
         "autofix",
         "autogenerated",
         "autolink",
+        "Autolinks",
         "autolinks",
```

Comment:
If the word in the dictionary is all-lowercase, then cspell is case-insensitive with regard to that word. E.g. if it has "autolinks", then "Autolinks" is not an error, but not the other way around. Please remove "Autolinks" and "ALGOLIA" if the dictionary already includes "autolinks" and "algolia".

Reply:
Okay...